test/ttmlir/Dialect/TTNN/simple_slice.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline %s| FileCheck %s
```

Comment:
Can we add TTIR and TTNN test cases to cover additional failures in verify methods? Take a look at matmul tests in TTIR and TTNN test dialect folders...